General Iteration (#322)
Roblox/creator-docs: content/en-us/education/battle-royale-series/ending-matches.md

   end
   ```

> 2. To find the player in the activePlayers table, use a for loop that iterates through the `activePlayer` table. Then, add an if statement that runs if a player matching the name passed into the function is found.

Reviewer: ```suggestion
2. To find the player in the `activePlayers` table, iterate through it using a `for` loop. Then, add an `if` statement that runs if a player matching the name passed into the function is found.
```
